ui/components/MesheryPerformance/PerformanceProfileTable.js

```diff
@@ -177,30 +178,35 @@ function MesheryTestProfiles({
         customBodyRender : function CustomBody(_, tableMeta) {
           return (
             <div>
-              <IconButton
-                style={iconMedium}
-                onClick={(ev) => {
-                  ev.stopPropagation();
-                  setSelectedProfile(testProfiles[tableMeta.rowIndex]);
-                }}
-                aria-label="edit"
-                // @ts-ignore
-                color="rgba(0, 0, 0, 0.54)"
-              >
-                <EditIcon style={iconMedium}/>
-              </IconButton>
-              <IconButton
-                style={iconMedium}
-                onClick={(ev) => {
-                  ev.stopPropagation();
-                  setSelectedProfile({ ...testProfiles[tableMeta.rowIndex], runTest : true });
-                }}
-                aria-label="run"
-                // @ts-ignore
-                color="rgba(0, 0, 0, 0.54)"
-              >
-                <PlayArrowIcon style={iconMedium} />
-              </IconButton>
+              <Tooltip title="Edit">
+                <IconButton
+                  style={iconMedium}
+                  onClick={(ev) => {
+                    ev.stopPropagation();
+                    setSelectedProfile(testProfiles[tableMeta.rowIndex]);
+                  }}
+                  aria-label="edit"
+                  // @ts-ignore
+                  color="rgba(0, 0, 0, 0.54)"
+                >
+                  <EditIcon style={iconMedium}/>
+                </IconButton>
+              </Tooltip>
```

Comment:
@sudhanshutech i think we can create a reusable tooltipIconButton component that is in MesheryPatterns.js and use it here also

Reply:
@sudhanshutech feasible?

Reply:
yes feasible, will make changes. Just missed notifications